### PR TITLE
[WFLY-14967] Move WildFly Preview to a native Jakarta namespace varia…

### PR DESCRIPTION
**Pull request by boris-unckel**
…nt of the jpa/eclipselink module

Fixes https://issues.redhat.com/browse/WFLY-14967

### CODE REVIEW

**Comment by boris-unckel** [comment]
@scottmarlow Could you review and approve this PR, please? It's the successor of #14855 implementing step 3 and 4 of https://github.com/wildfly/wildfly/pull/14855#issuecomment-962580936 It was not optional to use eclipselink 3.0.2 due to compile errors (javax.* vs. jakarta.*).

@bstansberry Due to splitting up the previous PR into several steps all your requirements https://github.com/wildfly/wildfly/pull/14855#discussion_r743993385 should be fulfilled now. Please have a look.

@yersan Thanks for your hint https://github.com/wildfly/wildfly/pull/14855#discussion_r772289876 regarding the correct place for the module/license. I have implemented it in this way, please have a look.

Thank you all for your support.

**Review by yersan — APPROVED** [review]
Changes look good to me

File: ee-9/source-transform/jpa/eclipselink/pom.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
```
Comment (scottmarlow) [inline ee-9/source-transform/jpa/eclipselink/pom.xml:4]: If this pr was created 3 days ago, the Copyright date could be 2022.

**Review by scottmarlow — CHANGES_REQUESTED** [review]
Please update copyright date in ee-9/source-transform/jpa/eclipselink/pom.xml  to 2022 or explain why its 2021.

**Review by scottmarlow — APPROVED** [review]

**Comment by scottmarlow** [comment]
Looks good to me, I couldn't hide my second comment above that is still visible but since the date was changed to 2022, I have no concerns now.  Thanks for the change @boris-unckel !

**Comment by bstansberry** [comment]
Thanks, @boris-unckel

**Pull request merged**